python_modules/libraries/dagster-aws/dagster_aws/ecs/launcher.py

```diff
@@ -50,13 +60,30 @@ def config_type(cls):
                 ),
             ),
             "container_name": Field(
-                dagster.String,
+                dagster.StringSource,
                 is_required=False,
                 default_value="run",
                 description=(
                     "The container name to use when launching new tasks. Defaults to 'run'."
                 ),
             ),
+            "secrets": Field(
+                dagster.Array(dagster.String),
+                is_required=False,
+                description=(
+                    "An array of AWS Secrets Manager secrets arns. These secrets will "
+                    "be mounted as environment variabls in the container."
+                ),
+            ),
+            "secrets_tag": Field(
+                dagster.StringSource,
```

Comment:
using 'dagster.Array' is less common than 'from dagster import Array' and then having just 'Array' here - not sure its a formal rule but I think its good to be consistent